pkg/theia/commands/policy_recommendation_retrieve_test.go

```diff
@@ -95,23 +96,74 @@ func TestPolicyRecommendationRetrieve(t *testing.T) {
 			expectedMsg:      []string{},
 			expectedErrorMsg: "error when getting policy recommendation job",
 		},
+		{
+			name:             "Unspecified name",
+			testServer:       httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {})),
+			nprName:          "pr-e292395c-3de1-11ed-b878-0242ac120001",
```

Comment:
we shouldn't hardcode name and use random ID all over the test cases

Reply:
we are testing it for the cases when we have the job id, we have to use this job id in order to retrieve

Reply:
you can randomly create job id in every test case, instead of hardcoding it in all the files